packages/graph-engine/src/nodes/series/fibonacci.ts

```diff
@@ -0,0 +1,116 @@
+import { INodeDefinition, ToInput, ToOutput } from '../../index.js';
+import { Node } from '../../programmatic/node.js';
+import { NumberSchema } from '../../schemas/index.js';
+import { arrayOf } from '../../schemas/utils.js';
+import { setToPrecision } from '../../utils/precision.js';
+
+type FibonacciValue = {
+	index: number;
+	value: number;
+};
+
+export default class NodeDefinition extends Node {
+	static title = 'Fibonacci Series';
+	static type = 'studio.tokens.series.fibonacci';
+	static description =
+		'Generates the Fibonacci sequence where each number is the sum of the previous two numbers';
+
+	declare inputs: ToInput<{
+		terms: number;
```

Comment:
This feels labelled weirdly, it feels like it should just be `length`

Reply:
agree